wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/CommandGroup.java

```diff
@@ -34,14 +34,14 @@
   /**
    * The commands in this group (stored in entries).
    */
-  private final Vector m_commands = new Vector();
+  private final List<Entry> m_commands = new ArrayList<>();
```

Comment:
I was under the impression that the entire library is single-threaded (with the exception of PID controllers, which I didn't touch)